packages/openapi-to-graphql/src/preprocessor.ts

```diff
@@ -21,6 +27,196 @@ import { GraphQLOperationType } from './types/graphql'
+          path,
+          method,
+          oas,
+          data,
+          options,
+          callbackObject
+        )
+
+        if (!responseSchema || typeof responseSchema !== 'object') {
+          handleWarning({
+            typeKey: 'MISSING_RESPONSE_SCHEMA',
+            message:
+              `Operation ${path} has no (valid) response schema. ` +
+              `You can use the fillEmptyResponses option to create a ` +
+              `placeholder schema`,
+            data,
+            log: preprocessingLog
+          })
+          continue
+        }
+
+        const responseDefinition = createDataDef(
+          responseSchemaNames,
+          responseSchema as SchemaObject,
+          false,
+          data,
+          undefined,
+          oas
+        )
+
+        // Parameters
+        const parameters = Oas3Tools.getParameters(
+          path,
+          method,
+          oas,
+          callbackObject
+        )
+
+        // Security protocols
+        const securityRequirements = options.viewer
+          ? Oas3Tools.getSecurityRequirements(
+              path,
+              method,
+              data.security,
+              oas,
+              callbackObject
+            )
+          : []
+
+        // Servers
+        const servers = Oas3Tools.getServers(path, method, oas, callbackObject)
+
+        // Whether to place this operation into an authentication viewer
+        const inViewer =
+          securityRequirements.length > 0 && data.options.viewer !== false
+
+        /**
+         * Whether the operation should be added as a Subscription field.
+         * By default, all operations containing callbacks are Subscription.
+         */
+        let isSubscription = true
+        let isMutation = false
+
+        const operation: Operation = {
+          operationId,
+          operationString,
+          description,
+          path,
+          method: method.toLowerCase(),
+          payloadContentType,
+          payloadDefinition,
+          payloadRequired,
+          responseContentType,
+          responseDefinition,
+          parameters,
+          securityRequirements,
+          servers,
+          callbacks: undefined,
+          inViewer,
+          isMutation,
+          isSubscription,
+          statusCode,
+          oas
+        }
+
+        /**
+         * Handle operationId property name collision
+         * May occur if multiple OAS are provided
+         */
+        if (operationId in data.operations) {
+          handleWarning({
+            typeKey: 'DUPLICATE_OPERATIONID',
+            message: `Multiple OASs share operations with the same operationId '${operationId}'`,
+            mitigationAddendum: `The operation from the OAS '${operation.oas.info.title}' will be ignored`,
+            data,
+            log: preprocessingLog
+          })
+        } else {
+          data.operations[operationId] = operation
+        }
+
+        operations.push(operation)
+      }
```

Comment:
The operations array is probably useless, but i was not sure if we might reuse it as reference for each callback later.

Reply:
Hmmm... I don't think we should push the `operation` to the `data.operations` or else we will add the operation as a `Query` or `Mutation` field.

Reply:
Each operation contained in callbacks is counted as `Subscription` because these operations are filtered later with `isSubscription`.
That's why i thought it might be useless to create this new array, unless we process them differently, but it didn't seem necessary.